### DISCUSSION TITLE
Seeking Methods to Track Loss, Complexity, and Score in PySR Iterations Without Custom SymbolicRegression.jl

### DISCUSSION THREAD

**Discussion by edwardaud** [MilesCranmer/PySR#597] [Q&A]
I am using PySR for symbolic regression and need a simpler way to log loss, complexity, and score per iteration without customizing SymbolicRegression.jl. Does anyone have a generalized approach or suggestions for this?

**Comment by MilesCranmer** [comment]
You can use this strategy: https://github.com/MilesCranmer/PySR/discussions/439#discussioncomment-7330778. But I would actually use more populations, and maybe 2 iterations per run so that migration happens.

And then when I finally finish adding TensorBoard, that would be even easier: https://github.com/MilesCranmer/SymbolicRegression.jl/pull/277

Also see progress on a UI in https://github.com/MilesCranmer/PySR/pull/589
